carma-messenger-core/cpp_message/src/MobilityPath_Message.cpp

```diff
@@ -45,7 +45,7 @@ namespace cpp_message
             uint64_t timestamp;
             //get sender id
             size_t str_len=message->value.choice.TestMessage02.header.hostStaticId.size;
-            if(str_len<=Header_constant.STATIC_ID_MAX_LENGTH && str_len!=0)
+            if(str_len<=Header_constant.STATIC_ID_MAX_LENGTH && str_len>=Header_constant.STATIC_ID_MIN_LENGTH)
```

Comment:
Is static ID variable in size? Or is it a fixed length in the encoding standard? This is fine if it's variable length, but padding may be required if it's fixed.

Reply:
Static ID is fixed, but the minimum length here is 2, If anything smaller than that is given as input, it assigns a default value of "[ ]"